.github/workflows/eslint.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
```

Comment:
vamos a eliminar este archivo y a hacer el pipeline manualmente, siguiendo este video:
https://youtu.be/sIhm4YOMK6Q hasta donde configura la parte de `eslint`.